src/hotspot/share/oops/generateOopMap.cpp

```diff
@@ -660,7 +660,7 @@ void GenerateOopMap::restore_state(BasicBlock *bb)
 }
 
 int GenerateOopMap::next_bb_start_pc(BasicBlock *bb) {
- int bbNum = bb - _basic_blocks + 1;
+ int bbNum = (int)(bb - _basic_blocks + 1);
```

Comment:
If this is an offset into a memory region then shouldn't it be `intptr_t` or `size_t`?

Reply:
I made bbNum an intptr_t.